packages/utils/fluid-main/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@fluid-experimental/fluid-main",
+  "version": "0.33.0",
+  "description": "The main entry point into Fluid Framework public packages",
+  "homepage": "https://fluidframework.com",
+  "repository": "https://github.com/microsoft/FluidFramework",
+  "license": "MIT",
+  "author": "Microsoft",
+  "sideEffects": false,
+  "main": "dist/index.js",
+  "module": "lib/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "concurrently npm:build:compile npm:lint",
+    "build:compile": "npm run tsc",
+    "build:full": "npm run build",
+    "build:full:compile": "npm run build:compile",
+    "clean": "rimraf dist *.tsbuildinfo *.build.log",
+    "eslint": "eslint --format stylish src",
+    "eslint:fix": "eslint --format stylish src --fix",
+    "lint": "npm run eslint",
+    "lint:fix": "npm run eslint:fix",
+    "tsc": "tsc",
+    "tsfmt": "tsfmt --verify",
+    "tsfmt:fix": "tsfmt --replace"
+  },
+  "dependencies": {
+    "@fluidframework/aqueduct": "^0.33.0",
+    "@fluidframework/core-interfaces": "^0.33.0",
+    "@fluidframework/datastore-definitions": "^0.33.0",
+    "@fluidframework/map": "^0.33.0",
+    "@fluidframework/merge-tree": "^0.33.0",
+    "@fluidframework/protocol-definitions": "^0.1017.1",
+    "@fluidframework/sequence": "^0.33.0",
+    "@fluidframework/shared-object-base": "^0.33.0",
+    "@fluidframework/view-interfaces": "^0.33.0"
+  },
+  "devDependencies": {
+    "@fluidframework/build-common": "^0.19.2",
+    "@fluidframework/eslint-config-fluid": "^0.21.0",
+    "@microsoft/api-extractor": "^7.7.2",
+    "@types/node": "^10.17.24",
+    "@typescript-eslint/eslint-plugin": "~4.2.0",
+    "@typescript-eslint/parser": "~4.2.0",
+    "concurrently": "^5.2.0",
+    "copyfiles": "^2.1.0",
+    "cross-env": "^7.0.2",
+    "eslint": "~7.9.0",
+    "eslint-plugin-eslint-comments": "~3.2.0",
+    "eslint-plugin-import": "~2.22.0",
+    "eslint-plugin-no-null": "~1.0.2",
+    "eslint-plugin-prefer-arrow": "~1.2.2",
+    "eslint-plugin-react": "~7.21.2",
+    "eslint-plugin-unicorn": "~22.0.0",
```

Comment:
I think you can remove these now that we are on eslint 7. That said if you leave them that's fine because I'll have to clean up all the other packages anyway.

Reply:
yeah, total copy/paste from other projects

Reply:
I was wrong about this! Leave them in.